### PR TITLE
cmake: fix path

### PR DESCRIPTION
**Pull request by stoffu**

### CODE REVIEW

**Comment by gnif** [comment]
This PR leaves much to be desired.

1. The build directory depends on where you decide to build bitmonero, there should be a seperate env variable for this, not a hard coded path based on the source directory.
2. Even if the above is done, the path is incorrect.
3. You have included linker changes, not just patch changes, that should be in a separate PR.

File: cnutil/CMakeLists.txt [hunk 1]
```diff
@@ -24,9 +26,11 @@ add_library(${CXXLIB} SHARED src/cnutil.cpp)
 target_link_libraries(${CXXLIB}
     cryptonote_core
     cryptonote_basic
-    crypto
+    cncrypto
```
Comment (gnif) [inline cnutil/CMakeLists.txt]: This is not needed here, only on the hashing library

File: CMakeLists.txt [hunk 2]
```diff
@@ -9,5 +9,7 @@ else()
     message("Monero surce directory is not defined, using default ${MONERO_DIR}")
 endif()
 
+include_directories(/usr/local/include)
```
Comment (gnif) [inline CMakeLists.txt]: This is clearly due to how you have built the project, it is incorrect to include system specifics like this.

File: cnutil/CMakeLists.txt [hunk 3]
```diff
@@ -11,11 +11,13 @@ include_directories("${MONERO_DIR}/src")
...
-    ${MONERO_DIR}/src/crypto
-    ${MONERO_DIR}/src/common
-    ${MONERO_DIR}/src/ringct
+    ${MONERO_DIR}/build/release/src/cryptonote_core
+    ${MONERO_DIR}/build/release/src/cryptonote_basic
+    ${MONERO_DIR}/build/release/src/crypto
+    ${MONERO_DIR}/build/release/src/common
+    ${MONERO_DIR}/build/release/src/ringct
+    ${MONERO_DIR}/build/release/contrib/epee/src
+    ${MONERO_DIR}/build/release/external/easylogging++
```
Comment (gnif) [inline cnutil/CMakeLists.txt]: Rather then altering the path which will break the build for everyone that uses cmake in the source tree, use an environment variable such as `MONERO_BUILD`.

**Comment by sammy007** [comment]
The path is correct as long as you follow build instructions in README. Now fixed for `v0.11.0.0`.

**Pull request closed without merging**